lib/pxenforcer.js

```diff
@@ -204,15 +212,21 @@ class PxEnforcer {
                     return callback(ScoreEvaluateAction.S2S_PASS_TRAFFIC);
                 }
 
-                this.logger.debug(`Risk API response returned successfully, risk score: ${ctx.score}, round_trip_time: ${ctx.riskRtt}ms`);
+                this.logger.debug(
+                    `Risk API response returned successfully, risk score: ${ctx.score}, round_trip_time: ${ctx.riskRtt}ms`
+                );
```

Comment:
Are all these types of changes necessary? I don't find them helpful but I don't feel strongly. (I'm assuming this was some auto-formatting thing that did this for you.)

Reply:
yap, it's Prettier's formatting.